homework/content.js

```diff
@@ -1,37 +1,42 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+function Content() {
+	this._wrapper = document.querySelector('#note-content-wrapper');
+}
 
-  function start() {
-    window.addEventListener('note-open', function(event) {
-      var note = event.detail;
-      resetWrapper();
-      drawNote(note);
-    });
-  }
+Content.prototype = {
 
-  function resetWrapper() {
-    _wrapper.innerHTML = '';
-  }
+	start : function () {
```

Comment:
Instead of using an anonymous function, set the listener as `this` and handle events with `handleEvent` method is better.